BMS Workspace-git/bms-1227-fw/Phantom_drivers/rtos/source/task_slave_pipeline.c

```diff
@@ -0,0 +1,170 @@
+/*
```

Comment:
@kkramer215 @lonamle This file should generally be how other tasks/modules will be designed. Let me know if you have any questions, comments, and/or concerns about this!